src/app/filters/filter-fields.component.scss

```diff
@@ -16,7 +16,7 @@
     background-color: #fff; //@color-pf-white;
     background-image: none;
     color: #8b8d8f; //@color-pf-black-500;
-    font-size: 12px;
+    //font-size: 12px;
```

Comment:
This looks like an older file. New code has been added for the type-ahead feature, which also adds another font-size property.

If you want to remove the font-size, you'll need to do it in two places -- in the latest file, that is.

  .btn-default {
    font-size: 12px; <<<<<<<<<<<<<<<<<<
  }
  .typeahead-input-container {
    position: relative;
    padding-right: 0px;
    .caret {
      position: absolute;
      z-index: 2;
      top: 10px;
      right: 5px;
    }
  }